File: ruby/URI_1046.rb
horas = gets.split

inicio = horas[0].to_i
fim = horas[1].to_i

if (inicio == fim) then
	duracao = 24
elsif (inicio > fim) then
	duracao = (fim + 24) - inicio
else
	duracao = fim - inicio
end

puts "O JOGO DUROU #{duracao} HORA(S)"

File: c-plus-plus/URI_1046.cpp
#include <iostream>

using namespace std;

int DuracaoJogo(int inicio, int fim) {
    if (inicio == fim) {
        return 24;
    } else if (inicio < fim) {
        return (fim - inicio);
    } else {
        return ((fim + 24) - inicio);
    }
}

int main() {
    int inicio, fim;
    cin >> inicio >> fim;
    cout << "O JOGO DUROU " << DuracaoJogo(inicio, fim) << " HORA(S)" << endl;
    return 0;
}
